.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: cpp
+
+matrix:
+  include:
+    - os: linux
+    - os: osx
+
+script:
+  - ./autogen.sh
+  - ./configure
+  - make
+  - sudo make install
```

Comment:
It's not necessary to `make install` in order to run tests. A better version of the script would be:

```
./autogen.sh
mkdir build
cd build
make check
```

Detailed test output is saved to `test/dotests.pl.log`

Reply:
I didn't put `make install` in for the tests but rather as a test of `make install` itself. Hopefully #12 suits this better version better. 🙂